sdk/identity/identity/review/identity.api.md

```diff
@@ -257,7 +257,9 @@ export const logger: AzureLogger;
 // @public
 export class ManagedIdentityCredential implements TokenCredential {
     constructor(clientId: string, options?: TokenCredentialOptions);
-    constructor(options?: ManagedIdentityCredentialOptions);
+    constructor(options?: ManagedIdentityCredentialClientIdOptions);
+    constructor(options?: ManagedIdentityCredentialResourceIdOptions);
+    constructor(options?: TokenCredentialOptions);
```

Comment:
Shouldn’t this be `ManagedIdentityCredentialOptions`?

Reply:
@sadasant I removed the type ManagedIdentityCredentialOptions since it was equivalent to TokenCredentialOptions.  Would you prefer that we type back and just extend TokenCredentialOptions and leave rest empty?

Reply:
That’s ok to me! Given that `ManagedIdentityCredentialOptions` was not part of our last GA. 👍